client/app/helpers/optionset.js

```diff
@@ -88,6 +88,7 @@ const OPTIONSET_LOOKUP = {
     dcpLegalinstrument: LANDUSE_FORM_OPTIONSETS.DCPLEGALINSTRUMENT,
     dcpRelatedacquisitionofpropertyopt: COMMON_OPTIONSETS.YES_NO_INTEGER,
     dcpOnlychangetheeliminationofamappedbutunimp: LANDUSE_FORM_OPTIONSETS.DCPONLYCHANGETHEELIMINATIONOFAMAPPEDBUTUNIMP,
+    dcpYesmappedbutunimprovedstreetelimated: LANDUSE_FORM_OPTIONSETS.DCPYESMAPPEDBUTUNIMPROVEDSTREETELIMATED,
```

Comment:
You should use this instead:

`dcpYesmappedbutunimprovedstreetelimated: COMMON_OPTIONSETS.YES_NO,`